lib/aims/realtime_util.py

```diff
@@ -89,6 +91,51 @@ def _pickle_filename(level_qc):
 
     return pickle_qc_file
 
+def delete_channel_id_from_pickle(level_qc, channel_id):
```

Comment:
This pickle file is not a config file. It's more like some sort of database containing information on what has already been downloaded or not. This file gets updated at each run of the script. Using pickle makes it easy to write and read from Python. It keeps the type of objects, var ...
But of course if you need to edit it manually, it would be easier to use a csv file. However, a PR would still be needed to stop the cron job.
And to be honest, a lot of the code was written to use this pickle file. So re-writting this to read and write a csv is not really my priority as it would affect SOOP TRV, FAIMMS and ANMN DAR YON BEAGLE

I also wrote ```delete_channel_id_from_pickle``` just in case for the future. I want to improve the README file as well to talk about this. This function would be used manually. Such as the ```delete_platform_entries_from_pickle``` function which is what I need to use to change the Beagle site code.